src/Home_excercise_5.java

```diff
@@ -0,0 +1,116 @@
+            System.out.println("Your text is Palindrome: " + isPalindrome(nextText));
+
+            Random random = new Random();
+            String [] secretWord = {"apple", "orange", "lemon", "banana", "apricot", "avocado", "broccoli", "carrot",
+                    "cherry", "garlic", "grape", "melon", "leak", "kiwi", "mango", "mushroom", "nut", "olive", " pea",
+                    "peanut", "pear", "pepper", "pineapple", "pumpkin", "potato"};
+            Scanner sc4 = new Scanner(System.in);
+
+            int indexElement = random.nextInt(secretWord.length);
+            String randomWord = secretWord[indexElement];
+
+            //System.out.println(indexElement);
+            //System.out.println(secretWord[indexElement]);
+            int account = 1;
+            while ((account-1)<randomWord.length()) {
+                System.out.println("Round "+account);
+                System.out.print("Enter your word (vegetable or fruit): ");
+                String yourRandomWord = sc4.nextLine().trim().toLowerCase();
+                if (yourRandomWord.equals(randomWord)) {
+                    System.out.println("Congratulation!!! You are Winner! ");
+                    break;
+                } else {
+                    System.out.println("Wrong! Try again!");
+                    account++;
+                    String checkWorld = "###############";
+                    System.out.println(randomWord.substring(0,(account-1))+checkWorld.substring(account-1));
+                }
+            }
+            System.out.println("Unfortunately, you are lose! Don't wary! Next time you get lucky!");
+        }
+
+
+        public static int findSymbolOccurance1(String yourString,String yourCharacter) {
+            int amountChracter = 0;
+            for (int i = 0; i < yourString.length(); i++) {
+                if ((String.valueOf(yourString.charAt(i))).equals(yourCharacter)){
+                    amountChracter++;
+                }
+            }
+
+            return amountChracter;
+
+        }
+        public static int findSymbolOccurance2(String yourString,String yourCharacter){
+            char yourChar = yourCharacter.charAt(0);
+            int numerChar = 0;
+            int count = 0;
+            int numer;
+
+            do {int indexC = yourString.indexOf(yourChar,numerChar); //It's every time == 0. Something wrong!
+                System.out.println(indexC);
+                numer = indexC;
+                if (indexC>0) count++; numerChar=indexC;
+            } while (numer!= -1);
+            return count;
+        }
+
+        public static int findWordPosition(String nextText, String checkText) {
+
+            int indexFirst;
+            if (nextText.contains(checkText) ) {
+                indexFirst = nextText.indexOf(checkText);
+                return indexFirst;
+            } return -1;
+        }
+        public static String stringReverse(String nextText) {
```

Comment:
Null validation should be added for an input string.
Please rename `nextText` to something more meaningful